src/test/rules/NoDirectEditorOptionsTest.ts

```diff
@@ -0,0 +1,47 @@
+import { RuleTester } from 'eslint';
+import { noDirectEditorOptions } from '../../main/ts/rules/NoDirectEditorOptions';
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('@typescript-eslint/parser'),
+  parserOptions: { sourceType: 'module' }
+});
+
+ruleTester.run('no-direct-editor-options', noDirectEditorOptions, {
+  valid: [
+    {
+      filename: 'src/main/ts/api/Options.ts',
```

Comment:
Should this also include the non api/ prefixed case that you have a comment about?

Reply:
Good point, that was something I added later and forgot to add tests for. They've now been added.